### DISCUSSION TITLE
More than one production systems

### DISCUSSION THREAD

**Discussion by BjBue** [pimcore/pimcore#14284] [Q&A / Support] [answered]
Hellotogether,

we havepimcore installed in an containerd system with azure.
There we have one production, one staging, one test and multi local systems.
The problem is that pimcore recognize more than one production systems, but that is not true. 
The APP_ENV is totally ignored.
So, please, what is the trick to make sure that only the production system is recognize as production system and all other not?

**Comment by mattamon** [comment, marked as answer]
Hey @BjBue,

the APP_ENV is not ignored but can be used with other configs.
How do you use your .env files?
Do you use e.g. also an .env.local? 

Please make sure that you also set the APP_DEBUG=true if you have a DEV system.

**Reply by BjBue** [reply]
Hello Mattamon, many many Thanks for your reply!

What do you mean with "how" do we use our .env-files?
And yes we use the .env.local.
In prod and stage we use the environment variables from the system. Is this the problem?

**Reply by mattamon** [reply]
Which server do you use? 
In the past I experienced some problems when using .env.local
Are you sure the .env.local is loaded correctly?

**Reply by BjBue** [reply]
Hello,
yes i am sure. 
It seems that the problem is solved by collegues.
My problem is that the "old" production system already recognized as prod systems. This old systems are offline but are already recognized. Do i need to talk to pimcore directly to make clear that this old systems are offline?

**Reply by mattamon** [reply]
Okay I am not entirely sure how your system is setup then. 

What seems to be the problem that the old system get recognized?

**Reply by BjBue** [reply]
Well, i am only the programmer. What i see is that in the licence of pimcore are 3 server listed but only one is active or online.
I do not understand why and i really do not know why there are 3 server listed or how pimcore recognize that there are 3 server.
Because i do not have enough knowledge about our infrastructure (it is an azure system) i poorly can not give you any more information how our system works, sorry for that.
So i would say our admins should ask there informations and we close this talk.
Many many thanks for your support :-)
